src/ImageRunModal.jsx

```diff
@@ -1089,6 +1165,44 @@
                 </Tabs>
             </Form>
         );
+
+        const cardFooter = () => {
+            let createRunText = _("Create and run");
+            let createText = _("Create");
+
+            if (this.props.prefill) {
+                createRunText = _("Clone and run");
+                createText = _("Clone");
+            }
+
+            return (
+                <>
+                    <Button variant='primary' id="create-image-create-run-btn" onClick={() => this.onCreateClicked(true)} isDisabled={(!image && selectedImage === "")}>
+                        {createRunText}
+                    </Button>
+                    <Button variant='secondary' id="create-image-create-btn" onClick={() => this.onCreateClicked(false)} isDisabled={(!image && selectedImage === "")}>
+                        {createText}
+                    </Button>
+                    <Button variant='link' className='btn-cancel' onClick={Dialogs.close}>
+                        {_("Cancel")}
+                    </Button>
+                </>
+            );
+        };
+
+        const modalTitle = () => {
+            let titleText = _("Create container");
+
+            if (this.props.prefill && this.props.pod)
+                titleText = _("Clone container in $0");
+            else if (this.props.prefill)
+                titleText = _("Clone container");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1270-20230621-082307-dd89f1ec-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)